src/components/views/avatars/BaseAvatar.tsx

```diff
@@ -108,119 +111,32 @@ const BaseAvatar: React.FC<IProps> = (props) => {
         url,
         urls,
         width = 40,
-        height = 40,
         resizeMethod = "crop", // eslint-disable-line @typescript-eslint/no-unused-vars
-        defaultToInitialLetter = true,
         onClick,
         inputRef,
         className,
+        type = "round",
         altText = _t("Avatar"),
-        ariaLabel = _t("Avatar"),
         ...otherProps
     } = props;
 
     const [imageUrl, onError] = useImageUrl({ url, urls });
 
-    if (!imageUrl && defaultToInitialLetter && name) {
-        const initialLetter = AvatarLogic.getInitialLetter(name);
-        const textNode = (
-            <span
-                className="mx_BaseAvatar_initial"
-                aria-hidden="true"
-                style={{
-                    fontSize: toPx(width * 0.65),
-                    width: toPx(width),
-                    lineHeight: toPx(height),
-                }}
-            >
-                {initialLetter}
-            </span>
-        );
-        const imgNode = (
-            <img
-                loading="lazy"
```

Comment:
Losing this seems bad

Reply:
Added to https://github.com/vector-im/compound-web/pull/57

Reply:
Added to https://github.com/vector-im/compound-web/pull/57